questions/javascript-hoisting-scope.md

```diff
@@ -0,0 +1,22 @@
+### In Javascript, Hoisting happens to Functions and Variable declared with `var`. Where can they be scoped?
+
+#### Answer
+
+Either Global or Function scoped.
+
+Unlike `let` or `const` they are not Block scoped and can only exist in one of two places because they are assigned a pointed before any evaluation of the code.
+
+#### Good to hear
+
+* There is more than one type of scope in Javascrit(Global, Function, Block)
+* Hoisting defines the references before actually evaluating the script
+
+##### Additional links
+
+<!-- Whenever possible, link a more detailed explanation. -->
+
+* [30 Seconds of Interviews: How does Hoisting work in Javascript](https://github.com/fejes713/30-seconds-of-interviews#how-does-hoisting-work-in-javascript)
+
+<!-- tags: (javascript) -->
+
+<!-- expertise: (1) -->
```

Comment:
This should be an "Easy" question I think

Reply:
Is 1 easy or medium? is our score 0 indexed? My bad

Reply:
yes, 0 indexed